recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+c_compiler:                    # [win]
+- vs2022                       # [win]
+cxx_compiler:                  # [win]
+- vs2022                       # [win]
```

Comment:
Windows uses Visual Studio 2019 by default

In case we are seeing a bug in Visual Studio 2019, this upgrades us to Visual Studio 2022